src/internals/bptree/node.rs

```diff
@@ -1,3 +1,5 @@
+// TODO: remove
```

Comment:
Still needed?

Reply:
Oops, forgot to do my dead code elimination pass. Fixed here: [Remove dead code; add back #[deny(dead_code)]](https://github.com/kanidm/concread/pull/125/commits/25420ba10b73d1f4a46b728a4ea54d6fc524d7ed).